### ISSUE TITLE
App name missing in `flyctl ssh console`

### ISSUE THREAD

**Issue by eylenburg** [pcrockett/mollysocket-fly#4]
When I tried `flyctl ssh console` I got an error saying the app name is missing. It worked with `flyctl ssh console -a mollysocket-fly` though.

To be fair, I tried this on another PC, not the one from where I deployed the app to fly.io. So I wasn't able do this from the folder where I cloned the git repo. Maybe that's why it failed.

**Comment by pcrockett** [comment]
Yup, that's why it failed. If you look closely at the configuration file on your original PC, you'll see the app name was generated for you. That's why there's a "template" file and copy step in the README:

```bash
cp fly.template.toml fly.toml
```

This is to prevent `flyctl` from editing the file that's in version control.

So yes, if you want to use this on another computer, you'll need to manually edit the config file on that computer to match.
